src/CampaignKit.WorldMap/Services/DefaultUserManagerService.cs

```diff
@@ -0,0 +1,89 @@
+﻿// <copyright file="DefaultUserManagerService.cs" company="Jochen Linnemann - IT-Service">
+// Copyright (c) 2017-2021 Jochen Linnemann, Cory Gill.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace CampaignKit.WorldMap.Services
+{
+    using System;
+    using System.Linq;
+    using System.Security.Claims;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+
+    /// <inheritdoc />
+    /// <summary>
+    ///     Class DefaultUserManagerService.
+    /// </summary>
+    /// <seealso cref="T:CampaignKit.WorldMap.Services.IUserManagerService" />
+    public class DefaultUserManagerService : IUserManagerService
+    {
+        /// <summary>
+        /// The application configuration.
+        /// </summary>
+        private readonly IConfiguration configuration;
+
+        /// <summary>
+        /// The application logging service.
+        /// </summary>
+        private readonly ILogger loggerService;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DefaultUserManagerService"/> class.
+        /// </summary>
+        /// <param name="configuration">The application configuration.</param>
+        /// <param name="loggerService">The logger service.</param>
+        public DefaultUserManagerService(IConfiguration configuration, ILogger<DefaultUserManagerService> loggerService)
+        {
+            this.configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+            this.loggerService = loggerService ?? throw new ArgumentNullException(nameof(loggerService));
+        }
+
+        /// <summary>
+        /// Gets the built in system user.
+        /// </summary>
+        /// <returns>
+        /// A ClaimsPrincipal object representing the system user.
+        /// </returns>
+        public ClaimsPrincipal GetSystemUser()
+        {
+            return new ClaimsPrincipal(
+                new ClaimsIdentity(
+                    new Claim[]
+                    {
+                        new Claim(ClaimTypes.NameIdentifier, "SystemAccount"),
```

Comment:
For what is this SystemAccount needed?

Reply:
During startup I check to see if the sample map has been added to the db yet.  If not, it creates the table entry and kicks of the process of making the tiles.  Inserting a record in the worldmapmaps table requires a user id.  (it's the partition id)  I created this mock identity to use for this particular case where the system is creating the map itself.  It's the only place where it is used.